### PR TITLE
1300 cleanup readme

### LINKED ISSUE #1300: Cleanup README
## Expected Behavior

The README documentation should enable a novice general user to run the software without understanding of how the software works.

## Actual Behavior

The README is too confusing.

### PR DESCRIPTION
**Pull request by spmcbride1201**
Resolves #1300

## How to Test

  - Proofread the README
  - Follow the Quick Start instructions and make sure it works as expected (I'll check the Windows instructions as home this evening)
  - Make sure links resolve

### CODE REVIEW

**Comment by tilleryd** [comment]
@spmcbride1201 Running `sed -i -e 's/__FABRIC_SERVER__/http:\/\/localhost:1337/g' public/index.html` led to the creation of a new file `public/index.html-e`.

File: README.md [hunk 1]
```diff
@@ -1,109 +1,109 @@
...
 
-#### `npm run build` to prepare the Dashboard for deployment to the core `gm-fabric-jvm` project
+If you are a developer interested in contributing to this project, please refer to our [Contribution Guide](./CONTRIBUTING.md).
 
-This builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+If you are a general user interested in exploring the features of this software, please refer to our [User Guide](./docs/overview.md) to learn more about the features of this software.
 
-Once built, the production bundle is minified and ready for deployment. The dashboard assumes that it is monitoring a microservice at the root path with Twitter Server metrics accessible at `/admin/metrics.json` and `/admin/threads`. The dashboard itself is served from `/gmadmin/`.
+If you are a systems administrator and with to install this software with live metrics and microservices, please refer to our externally-hosted [Grey Matter Fabric documentation site](http://www.deciphernow.com/)
```
Comment (tilleryd) [inline README.md]: typo, `with to install` should be `wish to install`

**Comment by tilleryd** [comment]
I like the Fabric logo 👍

File: README.md [hunk 2]
```diff
@@ -1,109 +1,109 @@
...
-### 2. Use docker to start an example Grey Matter Microservice
-
-### For a JVM-based microservice:
+<img alt="Animated GIF showing product in action" src="https://media.giphy.com/media/3o6fJ7w7GS3PyUKq2c/giphy.gif" >
 
-Run `docker run -it -p 1337:9990 spmcbride1201/gm-fab-jvm`
+## Quick Start
 
-After starting your microservice, you should see a valid JSON file [at this endpoint](http://localhost:1337/admin/metrics.json). If you see JSON data, you are ready to proceed.
+#### 1. If you do not already have Node.js 8.x LTS installed, [download and install it](https://nodejs.org/en/download/).
```
Comment (tilleryd) [inline README.md]: It would be nice to open this link in a new tab to keep the user on the README.

Comment (spmcbride1201) [reply README.md]: Oddly, the target attribute gets stripped off GitHub READMEs, so this isn't possible.

File: README.md (hunk 1 above)
Comment (spmcbride1201) [reply README.md]: Fixed!

**Pull request merged**